server/controllers/review.js

```diff
@@ -23,6 +25,23 @@ exports.createReview = asyncHandler(async (req, res, next) => {
   const review = await Review.create({ rating, text, profileReviewedId, reviewerId, requestId });
   await Profile.updateOne({ _id: profileReviewedId }, { $push: { reviewsReceived: review } });
 
+  const reviewerProfile = await Profile.findById(reviewerId);
+
+  const requesterFirstNameOrSomeone =
+    reviewerProfile && reviewerProfile.firstName ? reviewerProfile.firstName : 'someone';
```

Comment:
Sorry, Is this "someone" necessary, is firstName optional?

Reply:
yeah `firstName` is optional. 
I think right now if a new user signs up they would only have an email in their profile.
so It's possible to have firstName as `undefined`

Reply:
Great, _someone_ is definitely better than _undefined_,

Reply:
I would actually just prefer "Anonymous" than "someone." Bit more common in UI/UX